guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-for-uefi-http-boot-ipv6-provisioning_{context}"]
+= Configuring {Project} for UEFI HTTP Boot IPv6 Provisioning
+
+Use this procedure to configure {Project} to provision hosts in an IPv6 network with UEFI HTTP Boot provisioning.
+
+.Prerequisites
+
+* Ensure that your clients can access DHCP and TFTP servers.
+
+* Ensure that the UDP ports 67 and 68 are accessible by the client so that the client can send and receive a DHCP request and offer.
+
+* Ensure that the TCP port 8000 is open for the client to download files and Kickstart templates from {SmartProxy}.
+
+* Ensure that the host provisioning interface subnet has a TFTP {SmartProxy}, and Templates {SmartProxy} set. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html/provisioning_guide/configuring_networking#Configuring_Networking-Adding_a_Subnet_to_the_Satellite_Server[Adding a Subnet to {ProjectServer}] in the _Provisioning Guide_.
+
+.Procedure
+
+. Since clients that are booting from network cannot be identified by remote IPv6 address due to unmanaged DHCPv6 service, you must enable provisioning tokens. To enable provisioning tokens, navigate to *Administer* > *Settings* > *Provisioning* and set the *Token duration* setting to 0.
```

Comment:
I thought that setting the token duration to 0 disables them rather than enable. Note that this makes it insecure.

Reply:
Is there a way to provision securely? If not, let' be fair and provide more info about what exactly is insecure.

Reply:
Without provisioning tokens there is no authentication on provisioning templates, other than checking the IP address. These templates can include private data like passwords or other secrets. The attack would need to be able to spoof an IP on the target network so it's not trivial but it certainly lowers security.

Reply:
Your concern about security is valid but what I am saying is that we actually want users to ensure tokens are in use. Therefore it's more secure. In the original text I require the opposite (unsecure). So there's is nothing to worry about.